pkg/scheduler/framework/types.go

```diff
@@ -102,16 +107,17 @@ const (
 	//           - an existing Pod that was unscheduled but gets scheduled to a Node.
 	//
 	// Note that the Pod event type includes the events for the unscheduled Pod itself.
-	// i.e., when unscheduled Pods are updated, the scheduling queue checks with Pod/Update QueueingHint(s) whether the update may make the pods schedulable,
+	// i.e., when unscheduled Pods are updated, the scheduling queue checks with UnscheduledPod/Update QueueingHint(s) whether the update may make the pods schedulable,
 	// and requeues them to activeQ/backoffQ when at least one QueueingHint(s) return Queue.
-	// Plugins **have to** implement a QueueingHint for Pod/Update event
+	// Plugins **have to** implement a QueueingHint for UnscheduledPod/Update event
 	// if the rejection from them could be resolved by updating unscheduled Pods themselves.
 	// Example: Pods that require excessive resources may be rejected by the noderesources plugin,
 	// if this unscheduled pod is updated to require fewer resources,
 	// the previous rejection from noderesources plugin can be resolved.
-	// this plugin would implement QueueingHint for Pod/Update event
+	// this plugin would implement QueueingHint for UnscheduledPod/Update event
 	// that returns Queue when such label changes are made in unscheduled Pods.
-	Pod GVK = "Pod"
```

Comment:
Can we keep `Pod` as an alias of AssignedPod + UnscheduledPod (+ PodItself, if you agree with my comment) so that we can avoid breaking change?
So, when people subscribe `Pod` event, in [`buildQueueingHintMap`](https://github.com/kubernetes/kubernetes/blob/f0415a65b41e5eecc04425992c612832b27915f9/pkg/scheduler/scheduler.go#L390), we translate `Pod` event as `AssignedPod` + `UnscheduledPod` (+ `PodItself`).

Of course, either way, we should _strongly_ recommend selecting a specific Pod GVK for the requeueing efficiency in the release note though, at least, we can avoid breaking.

Reply:
> Can we keep Pod as an alias of AssignedPod + UnscheduledPod

This makes sense, I'll take a look at it.